<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiChannel</class>
 <widget class="QWidget" name="MultiChannel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1366</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1366</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MultiChannel</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1351</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;微软雅黑&quot;;
</string>
      </property>
      <property name="text">
       <string>端口:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="portEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;微软雅黑&quot;;
color: rgb(60, 68, 106);

background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="start">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
font: 11pt &quot;微软雅黑&quot;;

color: black; 
background-color:rgb(218, 223, 237);                                       

	
border:1px groove black; 
border-style: outset;
}
QPushButton::hover
{ /*鼠标悬停在按钮上时*/
font: 11pt &quot;微软雅黑&quot;;
color: black;
 background-color: rgb(255, 255, 255);
border:1px groove black; 
border-style: outset;
}
QPushButton::pressed
{
font: 11pt &quot;微软雅黑&quot;;
color: black; 
border:1px groove black; 
	background-color: rgb(201, 206, 218);
border-style: outset;
}

                   </string>
      </property>
      <property name="text">
       <string>启动</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Maximum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_30">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;微软雅黑&quot;;
</string>
      </property>
      <property name="text">
       <string>存储路径:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="currentPath">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;微软雅黑&quot;;
background-color: rgb(255, 255, 255);
color: rgb(60, 68, 106);
</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="setPath">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
font: 11pt &quot;微软雅黑&quot;;

color: black; 
background-color:rgb(218, 223, 237);                                       

	
border:1px groove black; 
border-style: outset;
}
QPushButton::hover
{ /*鼠标悬停在按钮上时*/
font: 11pt &quot;微软雅黑&quot;;
color: black;
 background-color: rgb(255, 255, 255);
border:1px groove black; 
border-style: outset;
}
QPushButton::pressed
{
font: 11pt &quot;微软雅黑&quot;;
color: black; 
border:1px groove black; 
	background-color: rgb(201, 206, 218);
border-style: outset;
}

                   </string>
      </property>
      <property name="text">
       <string>设置存储路径</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>1351</width>
     <height>711</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="spacing">
     <number>6</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="siglePB">
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
font: 11pt &quot;微软雅黑&quot;;

color: black; 
background-color:rgb(218, 223, 237);                                       

	
border:1px groove black; 
border-style: outset;
}
QPushButton::hover
{ /*鼠标悬停在按钮上时*/
font: 11pt &quot;微软雅黑&quot;;
color: black;
 background-color: rgb(255, 255, 255);
border:1px groove black; 
border-style: outset;
}
QPushButton::pressed
{
font: 11pt &quot;微软雅黑&quot;;
color: black; 
border:1px groove black; 
	background-color: rgb(201, 206, 218);
border-style: outset;
}

                   </string>
            </property>
            <property name="text">
             <string>单帧采集</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="multiPB">
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
font: 11pt &quot;微软雅黑&quot;;

color: black; 
background-color:rgb(218, 223, 237);                                       

	
border:1px groove black; 
border-style: outset;
}
QPushButton::hover
{ /*鼠标悬停在按钮上时*/
font: 11pt &quot;微软雅黑&quot;;
color: black;
 background-color: rgb(255, 255, 255);
border:1px groove black; 
border-style: outset;
}
QPushButton::pressed
{
font: 11pt &quot;微软雅黑&quot;;
color: black; 
border:1px groove black; 
	background-color: rgb(201, 206, 218);
border-style: outset;
}

                   </string>
            </property>
            <property name="text">
             <string>多帧采集</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="startPB">
            <property name="styleSheet">
             <string notr="true">QPushButton
{
font: 11pt &quot;微软雅黑&quot;;

color: black; 
background-color:rgb(218, 223, 237);                                       

	
border:1px groove black; 
border-style: outset;
}
QPushButton::hover
{ /*鼠标悬停在按钮上时*/
font: 11pt &quot;微软雅黑&quot;;
color: black;
 background-color: rgb(255, 255, 255);
border:1px groove black; 
border-style: outset;
}
QPushButton::pressed
{
font: 11pt &quot;微软雅黑&quot;;
color: black; 
border:1px groove black; 
	background-color: rgb(201, 206, 218);
border-style: outset;
}

                   </string>
            </property>
            <property name="text">
             <string>开始刺激</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stopPB">
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
font: 11pt &quot;微软雅黑&quot;;

color: black; 
background-color:rgb(218, 223, 237);                                       

	
border:1px groove black; 
border-style: outset;
}
QPushButton::hover
{ /*鼠标悬停在按钮上时*/
font: 11pt &quot;微软雅黑&quot;;
color: black;
 background-color: rgb(255, 255, 255);
border:1px groove black; 
border-style: outset;
}
QPushButton::pressed
{
font: 11pt &quot;微软雅黑&quot;;
color: black; 
border:1px groove black; 
	background-color: rgb(201, 206, 218);
border-style: outset;
}

                   </string>
            </property>
            <property name="text">
             <string>停止刺激</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="resetPB">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
font: 11pt &quot;微软雅黑&quot;;

color: black; 
background-color:rgb(218, 223, 237);                                       

	
border:1px groove black; 
border-style: outset;
}
QPushButton::hover
{ /*鼠标悬停在按钮上时*/
font: 11pt &quot;微软雅黑&quot;;
color: black;
 background-color: rgb(255, 255, 255);
border:1px groove black; 
border-style: outset;
}
QPushButton::pressed
{
font: 11pt &quot;微软雅黑&quot;;
color: black; 
border:1px groove black; 
	background-color: rgb(201, 206, 218);
border-style: outset;
}

                   </string>
            </property>
            <property name="text">
             <string>复位</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sendPB">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
font: 11pt &quot;微软雅黑&quot;;

color: black; 
background-color:rgb(218, 223, 237);                                       

	
border:1px groove black; 
border-style: outset;
}
QPushButton::hover
{ /*鼠标悬停在按钮上时*/
font: 11pt &quot;微软雅黑&quot;;
color: black;
 background-color: rgb(255, 255, 255);
border:1px groove black; 
border-style: outset;
}
QPushButton::pressed
{
font: 11pt &quot;微软雅黑&quot;;
color: black; 
border:1px groove black; 
	background-color: rgb(201, 206, 218);
border-style: outset;
}

                   </string>
            </property>
            <property name="text">
             <string>下发</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="sendText">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>26</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 11pt &quot;微软雅黑&quot;;

color: black; 
                                       
</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="hexCheck">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">
font: 11pt &quot;微软雅黑&quot;;

color: black; 

</string>
            </property>
            <property name="text">
             <string>HEX</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextBrowser" name="infoShow">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>450</width>
          <height>230</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>450</width>
          <height>230</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
font: 10pt &quot;微软雅黑&quot;;

color: black; 

</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>450</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>750</width>
        <height>600</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>891</width>
        <height>709</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;微软雅黑&quot;;

color: black; </string>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
